lte/gateway/c/core/oai/tasks/mme_app/mme_app_state_manager.cpp

```diff
@@ -32,7 +32,7 @@ constexpr char UE_ID_UE_CTXT_TABLE_NAME[] =
     "mme_app_mme_ue_s1ap_id_ue_context_htbl";
 constexpr char MME_IMSI2MME_UE_ID_MAP_NAME[] = "mme_imsi2ue_id_map";
 constexpr char MME_S11_TEID2MME_UE_ID_MAP_NAME[] = "mme_s11_teid2ue_id_map";
-constexpr char GUTI_UE_ID_TABLE_NAME[] = "mme_app_tun11_ue_context_htbl";
+constexpr char GUTI_UE_ID_MAP_NAME[] = "mme_app_guti_ue_context_map";
```

Comment:
This map has key as guti and data as mme_ue_s1ap_id, so this can be renamed as mme_app_guti2mme_ue_id_map